src/Faker/Provider/uk_UA/Company.php

```diff
@@ -25,6 +25,219 @@ class Company extends \Faker\Provider\Company
         'Мегамакс', 'Мульти', 'Майнер'
     );
 
+    /**
+     * @see list of Ukraine job title (2017-08-09), source: https://uk.wikipedia.org/wiki/%D0%A1%D0%BF%D0%B8%D1%81%D0%BE%D0%BA_%D0%BF%D1%80%D0%BE%D1%84%D0%B5%D1%81%D1%96%D0%B9
+     */
+    protected static $jobTitleFormat = array(
+        'Агроном',
```

Comment:
to save some vertical space, can you group the titles by line using the first letter?